packages/schema/package.json

```diff
@@ -6,8 +6,16 @@
   "private": true,
   "exports": {
     ".": {
+      "types": "./dist/src/index.types.d.ts",
+      "default": "./dist/src/index.types.js"
+    },
+    "./schemas": {
       "types": "./dist/src/index.d.ts",
       "default": "./dist/src/index.js"
+    },
```

Comment:
Any special reason why we have `.` and `./` both exported in here?

Reply:
`./*` could be removed actually. I kept it, just in case if we access schemas or types from a particular module's folder.

Reply:
Can you try removing it, building schema, and testing if it works in api-client package?

Reply:
Sure, it wont cause any issue in api-client